runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -221,4 +221,15 @@ default boolean claimSlime()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+			position = 16,
+			keyName = "swapClimb",
+			name = "Climb-Up",
+			description = "Swap climb with climb-up on staircases"
+	)

```

Comment:
I think IntelliJ doubles them by default, but make sure you only use *single* indents for code that's only one level deeper, like the ConfigItem above this one does.

If you look at the `Files changed` tab on this pull request, you can see where you've accidentally added some unnecessary padding to the Plugin file too.